Signal/src/AppDelegate.m

```diff
@@ -802,8 +803,8 @@ - (void)ensureRootViewController
         self.window.rootViewController = navigationController;
     } else {
         RegistrationViewController *viewController = [RegistrationViewController new];
-        UINavigationController *navigationController =
-            [[UINavigationController alloc] initWithRootViewController:viewController];
+        OWSNavigationController *navigationController =
+            [[OWSNavigationController alloc] initWithRootViewController:viewController];
```

Comment:
We need to use OWSNavigationController (or a subclass thereof) to support views that can cancel "navigate back".